lib/utils.pm

```diff
@@ -1289,18 +1289,22 @@ sub create_btrfs_subvolume {
   by providing  regex_to_find / text_to_replace as hash key/value pairs
 
   special key '--sed-modifier' allowing to add modifiers to expression
+  special key '--cat-file' allow to output full file content into serial. Disabled
+  by default
 =cut
 sub file_content_replace {
     my ($filename, %to_replace) = @_;
     $to_replace{'--sed-modifier'} //= '';
+    $to_replace{'--cat-file'} //= 0;
     my $sed_modifier = delete $to_replace{'--sed-modifier'};
+    my $cat_file = delete $to_replace{'--cat-file'};
     foreach my $key (keys %to_replace) {
         my $value = $to_replace{$key};
         $value =~ s/'/'"'"'/g;
         $key   =~ s/'/'"'"'/g;
         assert_script_run(sprintf("sed -E 's/%s/%s/%s' -i %s", $key, $value, $sed_modifier, $filename));
     }
-    script_run("cat $filename");
+    script_run("cat $filename") if $cat_file;
```

Comment:
I think if we do not wonna cut the file by default, we should just remove this line.
And if someone wonna cut it, he can call `script_run("cat $filename");` by him self.
But I do not see the point, why you wonne remove that cat? I think it is a nice debugging possibility and do not harm.

Reply:
reason to remove: cat'ing by default on every sed operation is spaming serial_terminal , also keep in mind that it is actually redundant because we saving most of file which we modifying in before/after tests actions 
don't see how forcing people to write extra function call is better then passing param into function which handles related flow

Reply:
I don't like that `--cat-file` option, because it sounds uncommon to me. I would like something like `--verbose` or `--debug` more.